staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/resource_access_test.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resourcequota
```

Comment:
This file was indeed added because previously, the LRU lookup was tested in [/kubernetes/plugin/pkg/admission/resourcequota/admission_test.go](https://github.com/kubernetes/kubernetes/blob/61774cd7176cae0c0324d23ab20e6c6b3038153f/plugin/pkg/admission/resourcequota/admission_test.go#L799) but due to the move out and the inaccessibility of the struct `quotaAccessor`, I added the LRU lookup test case to this new test file.

Reply:
in the admission_test.go, the test case is TestAdmitPodInNamespaceWithoutQuota? what about adding the access to quotaAccessor just for the testing purpose? then you may keep the test case there, instead of adding a new one here.

Reply:
ok. so this is testing the cache hit case, probably better to add a cache miss case too.